### DISCUSSION TITLE
Question: Is this repo up to date?

### DISCUSSION THREAD

**Discussion by gius** [liquidprojections/LiquidProjections#140] [Q&A] [answered]
Hi, I wanted to ask if this repository is still maintained. Do you use it for your production projects?
Thank you

**Comment by dennisdoomen** [comment, marked as answer]
It has been used in production for several years, but it's mostly feature complete.

**Reply by gius** [reply]
Thank you for the quick answer! 

I am going to start a new project and do some investigation about the tech stack. Are there any recommendations you could share? I would like to avoid reinventing the wheel. Marten looks great and well-documented as well, but I would like to avoid PostgreSQL.

**Reply by dennisdoomen** [reply]
Did you check out https://eventuous.dev/?

**Reply by gius** [reply]
Oh, thank you, I'll have a look! I had just Marten, NEventStore, EventFlow (+Liquid Projections) on my list. Marten looks really good and actively developed, but the rest seemed a little bit abandoned to me.

**Reply by dennisdoomen** [reply]
Your conclusion is correct.
